mariadb/templates/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Name }}"
```

Comment:
@sameersbn did you mean for this to be "{{ .Chart.Name }}-{{ .Chart.Version }}" ?